src/app.js

```diff
@@ -27,9 +40,42 @@ const quoteData = [
 
 $(document).ready(function() {
   const quotes = new QuoteList(quoteData);
+
   const simulator = new Simulator({
     quotes: quotes,
   });
 
+  console.log(quotes);
+  const quoteListView = new QuoteListView({
+    model: quotes,
+    template: _.template($('#quote-template').html()),
+    tradeTemplate: _.template($('#trade-template').html()),
+    el: 'main',
```

Comment:
This is a bit of a problem, it leads your `QuoteListView` instance and `OrderListView` instance to overlap on the DOM, which means they could interfere with each other.  

Best practices in Backbone is for independent views to **NOT** overlap in the DOM.  You can have subviews, but views that don't know about each other (one isn't managing the other) should not overlap.